docs/faq.rst

```diff
@@ -31,8 +34,34 @@ the v3.2 code, with the goal of revisiting it when ``tedana`` is more stable.
 
 Anyone interested in using v3.2 may compile and install an earlier release (<=0.0.4) of ``tedana``.
 
+What is the warning about ``duecredit``?
+`````````````````````````````````````````
+``duecredit`` is a python package that is used, but not required by ``tedana``. These warnings do
+not affect any of the processing within the ``tedana``. To avoide this warning, you can install
+``duecredit`` with ``pip install duecredit``. For more information about ``duecredit`` and concerns about 
+the citation and visibility of software or methods, visit the `duecredit`_ github. 
+
+.. _duecredit: https://github.com/duecredit/duecredit
+
 .. _here: https://bitbucket.org/prantikk/me-ica/commits/906bd1f6db7041f88cd0efcac8a74074d673f4f5
 
 .. _NeuroStars: https://neurostars.org
 .. _fMRIPrep: https://fmriprep.readthedocs.io
 .. _afni_proc.py: https://afni.nimh.nih.gov/pub/dist/doc/program_help/afni_proc.py.html
+
+Multi-echo fMRI
+---------------
+
+Will I encounter SAR limits more often with multi-echo fMRI?
```

Comment:
Can we define SAR ?

Reply:
I have tried to navigate this area without making it sound like we cook participants in the scanner. Linked out to a more thorough explanation.